ghost/core/core/server/data/migrations/versions/5.57/2023-07-26-12-44-stripe-products-nullable-product.js

```diff
@@ -0,0 +1,4 @@
+const {createSetNullableMigration} = require('../../utils');
+
+// We need to disable foreign key checks because if MySQL is missing the STRICT_TRANS_TABLES mode, we cannot revert the migration
+module.exports = createSetNullableMigration('stripe_products', 'product_id', {disableForeignKeyChecks: true});
```

Comment:
Flagging that the tech spec documents this change, but doesn't have any discussion of pros/cons and in particular if there's any fallout work that would need to be done in any other services to maintain data consistency?

Reply:
This was discussed and I've raised the issue again, but still think this is the best solution overall 😊